chapter_preliminaries/linear-algebra.md

```diff
@@ -777,12 +773,30 @@ to describe the most intensive calculations
 required when computing each layer in a neural network
 given the values of the previous layer.
 
+:begin_tab:`mxnet`
 Expressing matrix-vector products in code with tensors,
 we use the same `dot` function as for dot products.
 When we call `np.dot(A, x)` with a matrix `A` and a vector `x`,
 the matrix-vector product is performed.
 Note that the column dimension of `A` (its length along axis 1)
 must be the same as the dimension of `x` (its length).
+:end_tab:
+
+:begin_tab:`pytorch`
+Expressing matrix-vector products in code with tensors, we use
+the `mv` function. When we call `torch.mv(A, x)` with a matrix
+`A` and a vector `x`, the matrix-vector product is performed.
+Note that the column dimension of `A` (its length along dim 1)
```

Comment:
dim -> axis.
Besides, even using "dim", we can spell it out as "dimension" in proper English :)

Reply:
Done!